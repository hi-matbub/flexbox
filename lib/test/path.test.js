import React from 'react';
import ReactDOM from 'react-dom';
import {BrowserRouter as Router} from 'react-router-dom';

import renderer from 'react-test-renderer';
import Path from '../comps/path_.jsx';

it('renders smoke test without crashing', () => {
  const div = document.createElement('div');
  ReactDOM.render(
    <Router>
      <Path />
    </Router>,
    div,
  );
  ReactDOM.unmountComponentAtNode(div);
});

it('renders snapshot as expected', () => {
  const tree = renderer
    .create(
      <Router>
        <Path />
      </Router>,
    )
    .toJSON();
  expect(tree).toMatchSnapshot();
});
