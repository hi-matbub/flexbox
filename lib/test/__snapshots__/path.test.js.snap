// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders snapshot as expected 1`] = `
<a
  href=""
  onClick={[Function]}
  style={
    Object {
      "color": "#000",
      "letterSpacing": "1px",
      "lineHeight": "28px",
    }
  }
>
   
</a>
`;
